apps/mull-ui/src/app/app.tsx

```diff
@@ -117,6 +117,7 @@ export const App = () => {
             </ChatPagesHeader>
           </PrivateRoute>
           <PrivateRoute path={ROUTES.CAMERA} component={TrashRecognitionPage} />
+          <PrivateRoute path={ROUTES.DIRECT_MESSAGES} component={DirectMessagePage} />
```

Comment:
Just to be clear, this is temporary right?

Reply:
Yep! This is temporary